cirque/common/cirqueprompt.py

```diff
@@ -0,0 +1,122 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import cmd
+import pprint
+import argparse
+
+from cirque.home.home import CirqueHome
+
+HOME = None
+pp = pprint.PrettyPrinter(indent=2)
+
+
+class CirquePrompt(cmd.Cmd):
+    prompt = 'cirque> '
```

Comment:
please add help function
please add list homes
please add list devices
please add send_cmd which can send command to particular device.

Reply:
is there any chances that developer would create multiple homes locally?
help function, ack!
home_devces = list devices
run_exec = send_cmd